Azure/azure-cli
# [AppService]  BREAKING CHANGE: Add support for az staticwebapp show; az staticwebapp create; az staticwebapp update

**Description**
1. az staticwebapp browse -> az staticwebapp show
2. add option of sku for az staticwebapp create --sku
3. add command of az staticwebapp update

**Testing Guide**

Tested commands - working as expected:
     1. az staticwebapp show -n my-first-static-web-app
     2. az staticwebapp create -n my-second-static-web-app -g rg -s https://github.com/jiansong-msft/my-first-static-web-app -l CentralUS -b main --token github-token --sku Free
     3. az staticwebapp update -n my-second-static-web-app --tags "key=value"
     4. az staticwebapp update -n static-app-not-exists throws
     5. with invalid sku options passed in throws

---

This checklist is used to make sure that common guidelines for a pull request are followed.
